### PR TITLE
feat: Added Playlist

### PR DESCRIPTION
**Pull request by mnath30**
Added the following:
1. Playlist Page to see the list of all playlists available.
2. Single playlist page to see the videos added in the playlist
3. Create Playlist
4. Add to playlist
5. Remove from playlist
6. API calls for playlists

User will be able to add the videos to the playlist when logged in and also remove the videos from the playlist. Playlist page will display all the playlists created by the user. User will be able to navigate and see the videos added to any playlist by clicking on the playlist name from the playlist page.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="modal"></div>
+    <div id="overlay"></div>
```
Comment (omjadhav85) [inline public/index.html:21]: not sure if its adivisable to add content outside "root" element. You can add the Modal component at the end of your playlist page and toggle it based on a state.

**Review by omjadhav85 — APPROVED** [review]
All functionalities working as expected. Can implement suggested changes regarding Modal implementation. Else looks good to merge!

**Review by prathmeshjagtap — APPROVED** [review]
The code is  Clean, Just add the cursor pointer on the checkbox while adding to the playlist.
All functionality is working as expected

File: public/index.html (hunk 1 above)
Comment (mnath30) [reply public/index.html:21]: Thankyou for your comments.
I have used the above approach in the other projects, wanted to use react portal for this application.

**Pull request merged**